templates/history-downloads.php

```diff
@@ -9,23 +9,27 @@
  *            Checks status on individual order items when determining download link visibility.
  */
 
-if( ! empty( $_GET['edd-verify-success'] ) ) : ?>
-<p class="edd-account-verified edd_success">
-	<?php _e( 'Your account has been successfully verified!', 'easy-digital-downloads' ); ?>
-</p>
-<?php
+if ( ! empty( $_GET['edd-verify-success'] ) ) : ?>
+	<p class="edd-account-verified edd_success">
+		<?php esc_html_e( 'Your account has been successfully verified!', 'easy-digital-downloads' ); ?>
+	</p>
+	<?php
 endif;
 /**
  * This template is used to display the download history of the current user.
  */
 $customer = edd_get_customer_by( 'user_id', get_current_user_id() );
+$page     = get_query_var( 'paged' ) ? get_query_var( 'paged' ) : 0;
```

Comment:
```suggestion
$page     = get_query_var( 'paged' ) ? get_query_var( 'paged' ) : 1;
```

This makes the offset logic better, IMO. (see associated `edd_get_orders()` suggestion)